src/administrative-sdk/speech-challenge/speech-challenge.js

```diff
@@ -9,8 +9,12 @@ export default class SpeechChallenge {
    * @param {?string} topic - A question or topic serving as guidance.
    * @param {?string} referenceAudioUrl - The reference audio fragment URL. If one is not yet available or audio is
    * not yet registered to the challenge it can be set to 'null'.
+   * @param {?string} srtUrl - URL of a possible .srt file to accompany this challenge.
+   * @param {?string} imageUrl - URL of a possible image file to accompany this challenge.
+   * @throws {Error} srtUrl parameter of type "string|null" is required.
+   * @throws {Error} imageUrl parameter of type "string|null" is required.
    */
-  constructor(id, topic, referenceAudioUrl) {
+  constructor(id, topic, referenceAudioUrl, srtUrl = null, imageUrl = null) {
```

Comment:
What happens to regular object construction then?

`const org = new Organisation('identifier', 'orgname');`

turns into?

`const org = new Organisation({id: 'identifier', name: 'orgname'});`

Is constructor overloading a thing? Don't think so...